homeworks/HW_05/task2.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+let averageGrade = 71;
+
+switch (true) {
+    case (averageGrade <= 60):
```

Comment:
За умовою незадовліьно не включає 60
```suggestion
    case (averageGrade < 60):
```